specs/codecs/dag-eth/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Specs: Ethereum"
-navTitle: "Ethereum"
+title: "Specs: DAG-ETH"
+navTitle: "DAG-ETH"
```

Comment:
```suggestion
navTitle: "DAG-ETH"
eleventyNavigation:
  synopsys: "Specification for Ethereum as an IPLD Data Structure and the suite of codecs used to convert to and from the IPLD Data Model"
```

.. or adapt wording as you think appropriate - we don't have enough of these synopsys' on the site yet and "DAG-ETH" is now less descriptive than just "Ethereum". The synopsys shows up on the index page that links to each codec spec.